orchestrator-salt/src/main/resources/salt/salt/nginx/init.sls

```diff
@@ -1,3 +1,24 @@
+{% set gateway = salt['pillar.get']('gateway') %}
+
+/etc/nginx/sites-enabled/ssl.conf:
+  file.managed:
+    - makedirs: True
+    - source: salt://nginx/conf/ssl.conf
+
+# when gateway is defined, we do NOT config user facing cert and ssl for port 443, because services are available through knox on 8443
```

Comment:
I think we shall overwrite the ssl.conf since the image burning still contains such ssl.conf which listens on 443

Reply:
it is still overwritten at line 3